src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+abstract class Figure {
+    protected Color color;
+
+    public void setColor(Color color) {
+        this.color = color;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    public abstract double area();

```

Comment:
Don't use abstract classes to set behavior for classes. Consider whether `Figure` should be an abstract class or an interface based on its use case in this context.